design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -185,6 +215,7 @@ type NegotiableQuote {
     # attachments: [AttachmentContent]
     comments: [NegotiableQuoteComment!]
     history: [NegotiableQuoteHistoryEntry!]
+    billing_address: BillingCartAddress
```

Comment:
We need a Negotiable Quote specific version of this
`NegotiableQuoteAddressInterface` -> `NegotiableQuoteBillingAddress` -> `NegotiableQuoteAddressRegion`/`NegotiableQuoteAddressCountry`